Source/bmalloc/bmalloc/Allocator.cpp

```diff
@@ -42,7 +42,7 @@ Allocator::Allocator(Heap& heap, Deallocator& deallocator)
     : m_heap(heap)
     , m_deallocator(deallocator)
 {
-    BASSERT(!Environment::get()->isSystemHeapEnabled());
+    BASSERT(!Environment::get()->shouldBmallocAllocateThroughSystemHeap());
```

Comment:
Weird, `Bmalloc` is not how I would have CamelCased `bmalloc` but it appears consistent.